app/views/components/inputCurrency.scala.html

```diff
@@ -67,7 +67,7 @@
 
 @inputSnippet(describedBy: String) = {
 @defining {
-    id="@id" name="@name" type="text" inputmode="decimal" @form(s"$id").value.map { value => value="@value" }@if(describedBy.nonEmpty) { aria-describedby="@describedBy"}
+    id="@id" name="@name" type="text" inputmode="decimal" @form(s"$id").value.map { value => value="@value" }@if(describedBy.nonEmpty) { aria-describedby="@id"}
```

Comment:
should this really be set to 'id'? is there any point to having 'describedBy' in that case?

Reply:
I fixed the wave issue by setting it to id.